website/docs/cloud-docs/workspaces/variables/index.mdx

```diff
@@ -121,22 +132,25 @@ Variables in the `.auto.tfvars` files take precedence over variables in the `ter
 
 Consider an example workspace that has the following variables applied:
 
-| Source                | Scope             |  ACCESS_KEY | ACCESS_ID  | VAR1 | KEY1 | VAR2 | replicas |
-| --------------------- | ------------------ | ---------- | ---------- | ---- | ---- | ---- | -------- |
-| Command Line Argument | Run-Specific       |            |            |      |      |      | `9`      |
-| Local Environment     | Run-Specific       |           |            |      |      |      | `8`      |
-| Variables             | Workspace-Specific | `g47fh474` | `874hf7u4` | `h`  |      |      | `1`      |
-| A_Variable_Set        | Non-Global       |              |            | `y`  | `x`  |      | `2`      |
-| B_Variable_Set        | Global          |               |            |      | `z`  | `a`  | `3`      |
+| Source                | Scope                  | ACCESS_KEY | ACCESS_ID  | VAR1 | KEY1 | VAR2 | KEY2 | replicas |
+| --------------------- | ---------------------- | ---------- | ---------- | ---- | ---- | ---- | ---- | -------- |
+| Command Line Argument | Run-Specific           |            |            |      |      |      |      | `9`      |
+| Local Environment     | Run-Specific           |            |            |      |      |      |      | `8`      |
+| Variables             | Workspace-Specific     | `g47fh474` | `874hf7u4` | `h`  |      |      |      | `1`      |
```

Comment:
To keep the picture in the diagram in sync, I'd suggest moving this ACCESS_KEY & ACCESS_ID to the B_Variable_Set row. And then the ACCESS_KEY & ACCESS_ID mentioned in line 136/148 to the line with **B_Variable_Set**

Reply:
Yes! Thank you for catching this 👁️ I will update.